spec/haskell/src/SEL4/Object/Reply.lhs

```diff
@@ -188,6 +195,12 @@ on the thread state of the replyTCB of the replyPtr
 > getReplyTCB r = liftM replyTCB (getReply r)
 
 > setReplyTCB :: Maybe (PPtr TCB) -> PPtr Reply -> Kernel ()
-> setReplyTCB tptrOpt rptr = do
->     r <- getReply rptr
->     setReply rptr (r { replyTCB = tptrOpt})
+> setReplyTCB tptrOpt rptr = updateReply rptr (\reply -> reply { replyTCB = tptrOpt })
+
+> replyClear :: PPtr Reply -> PPtr TCB -> Kernel ()
```

Comment:
Hmm. I think this is a duplicated old version. Maybe something went wrong when you rebased. I think Corey moved this to `ObjectType.lhs` and it should have `cancelIPC` instead of `replyUnlink`.

Reply:
Good spotting! As far as I can tell we should just be able to delete this duplicated version. It's not exported so I don't think it's accidentally used anywhere.

Reply:
Alright, as expected, removing this didn't break anything On My Machine™️ . Gonna start a testboard anyway, just in case.